app/views/courses/_content_status_tag.html.erb

```diff
@@ -1 +1,25 @@
-<span class="govuk-tag phase-tag--small phase-tag--published" style="border: 3px solid red">Published</span>
+<%
```

Comment:
How do you feel about moving this to a helper?

Reply:
Is it likely to be used anywhere else?

Reply:
If you prefer it I'll do it though sure

Reply:
100% a view helper

Reply:
A part of me wants to get Draper in here to handle this kind of display logic.

Reply:
just use this as inspiration 

https://github.com/DFE-Digital/manage-courses-ui/blob/master/src/ui/Helpers/ViewModelHelpers.cs

😆

Reply:
It's for separation of concerns and keeping the view clean, rather than reusability 🐱 Logic in templates is considered harmful, hacker news said it

Reply:
React has proven that logic in templates is alright, but, this is Rails-landia, and I'm happy to refactor this. 👍

Reply:
Refactored, check it out y'all